scripts/core/INTRO.rst

```diff
@@ -234,6 +234,28 @@ An example of an environment variable for setting up the null adapter library wi
 
   UR_LOG_NULL="level:warning;output:stdout"
 
+Adapter Discovery
+---------------------
+UR is capable of discovering adapter libraries in the following ways in the listed order:
+
+  - Check paths to the adapters set in `UR_ADAPTERS_FORCE_LOAD` environment variable.
+
+    + All other adapter discovery methods are disabled when this environment variable is used.
+
+  - Search directories specified in `UR_ADAPTERS_SEARCH_PATH` environment variable.
+
+  - Leave adapter discovery for the OS.
```

Comment:
- If on Linux, use the shared library discovery mechanism (see `ld.so(8)` for details).

Reply:
Done.